src/components/Amount.tsx

```diff
@@ -50,11 +53,11 @@ export default function Amount({
       {...rest}
     >
       {asset && (
-        <Currency amount={asset.amount} ticker={asset.ticker} icon={true} uuid={asset.uuid} />
+        <Currency amount={asset.amount} isNFTAsset={isNFTAsset} ticker={asset.ticker} icon={true} uuid={asset.uuid} decimals={getDecimals(asset.amount)}/>
```

Comment:
Why are we changing the decimals thing?

Reply:
removed. was useful in testing alternative decimal display models.